tests/Unit/Emogrifier/HtmlProcessor/CssInlinerTest.php

```diff
@@ -1223,6 +1223,30 @@ public function emogrifyByDefaultIgnoresInvalidCssSelectors()
         static::assertContains('background-color: blue', $html);
     }
 
+    /**
+     * @param string[][] $datasetsWithSpaces
+     *
+     * @return string[][] Datasets with spaces in the first array entry of each replaced with various alternative
+     *                    whitespace.  Keys will have " with ..." appended to provide an appropriate description.
+     */
+    private function getDatasetsWithVaryingWhitespace(array $datasetsWithSpaces)
+    {
+        $spaceReplacements = [
+            'extra spaces' => '  ',
+            'linefeeds' => "\n",
+            'Windows line endings' => "\r\n",
+        ];
+        $datasets = [];
+        foreach ($datasetsWithSpaces as $description => $dataset) {
+            foreach ($spaceReplacements as $spaceReplacementDescription => $spaceReplacement) {
+                $datasets[$description . ' with ' . $spaceReplacementDescription] = [
+                    0 => str_replace(' ', $spaceReplacement, $dataset[0]),
```

Comment:
You don't need the array key 0 here as this is the default key for the first element AFAIK.